src/content/docs/fr/tutorial/0-introduction/index.mdx

```diff
@@ -0,0 +1,39 @@
+---
+type: tutorial
+unitTitle: 'Bonjour, le monde!'
+title: Realisez votre premier Blog Astro
+description: >-
+  Apprendre les bases d'Astro a l'aide d'un tutoriel base projet. Toutes les connaissances
+  dont vous avez besoin pour commencer!
+i18nReady: true
+---
+import Badge from '~/components/Badge.astro';
+import Checklist from '~/components/Checklist.astro';
+import Box from '~/components/tutorial/Box.astro';
+import Lede from '~/components/tutorial/Lede.astro';
+
+
+<Lede> Dans de tutoriel, vous apprendrez les fonctionnalites cles d'Astro en realisant un blog totalement fonctionnel, de zero jusqu'au deploiement en ligne! 🚀 </Lede>
+
+Sur votre chemin, vous: 
+- Configurerez un environnement de developement 
+- Creerez des pages et des articles de blog posts pour votre site web
+- Construirez avec des composants Astro
+- Chercherez et travaillerez avec des fichiers locaux
+- Ajouterez de l'interactivite sur votre site
+- Deploierez votre site sur internet
+
+Voulez vous avoir un percu de ce que vous allez realiser? Vous pouvez voir le projet fini sur [GitHub](https://github.com/withastro/blog-tutorial-demo) ou [StackBlitz](https://stackblitz.com/github/withastro/blog-tutorial-demo/tree/complete?file=src%2Fpages%2Findex.astro).
+
+:::note
+Si vous preferez commencer a explorer Astro avec une version pre-construite, vous pouvez visiter https://astro.new et selectionner un gabarit de demmarrage, l'ouvrir et l'editer dans l'editeur en ligne.
```

Comment:
I suggest for line 29 => Si vous souhaitez plutôt commencer à explorer Astro avec un site Astro pré-construit, vous pouvez visiter https://astro.new et choisir un modèle, l'ouvrir et le modifier dans un éditeur en ligne.